dev/system-test/firestore.ts

```diff
@@ -288,7 +289,7 @@ describe('DocumentReference class', () => {
       })
       .then(doc => {
         const data = doc.data()!;
-        expect(data.pathValue.path).to.equal(
+        expect((data.pathValue as DocumentReference).path).to.equal(
```

Comment:
This should probably return a DocumentReference<unknown> and needs to be fixed in the Serializer (serializer.ts, line 55)

Reply:
Hmm, I removed the cast to DocumentReference and it still works fine. I think I probably forgot to remove it.